TESTING.md

```diff
@@ -0,0 +1,38 @@
+# How to run phpunit tests
+
+For reference: https://propelorm.org/Propel/cookbook/working-with-unit-tests.html
+
+Use separate docker containers to run tests, because this package requires own database to test its functionality.
+
+### Build and start containers
+
+```shell
+docker-compose up
+```
+
+### Install dependencies
+```shell
+docker-compose exec php bash -c "composer install"
+```
+
+### Prepare test databases
+```shell
+docker-compose exec mysql sh -c "mysql -u root -proot < /var/www/test/init_db.sql"
+```
+
+### Prepare tests
+```shell
+docker-compose exec php bash -c "/var/www/test/reset_tests.sh"
+```
+
+Note that getting errors like these is OK:
+```
+BUILD FAILED
+Propel/generator/build.xml:95:15: No project directory specified
+```
+
+### Run tests:
+
+```bash
+docker-compose exec php bash -c "vendor/bin/phpunit"
```

Comment:
I'm getting this error also when executing this command probably due to previous error
```
Warning: include(/var/www/test/tools/helpers/bookstore/../../../fixtures/bookstore/build/conf/bookstore-conf.php): failed to open stream: No such file or directory in /var/www/runtime/lib/Propel.php on line 302

Warning: include(): Failed opening '/var/www/test/tools/helpers/bookstore/../../../fixtures/bookstore/build/conf/bookstore-conf.php' for inclusion (include_path='/var/www/runtime/lib:/var/www/generator/lib:/var/www/vendor/pear-pear.php.net/Archive_Tar:/var/www/vendor/pear-pear.php.net/Console_Getopt:/var/www/vendor/pear-pear.php.net/PEAR:/var/www/vendor/pear-pear.php.net/PEAR_PackageFileManager2:/var/www/vendor/pear-pear.php.net/PEAR_PackageFileManager_Plugins:/var/www/vendor/pear-pear.php.net/Structures_Graph:/var/www/vendor/pear-pear.php.net/XML_Parser:/var/www/vendor/pear-pear.php.net/XML_Serializer:/var/www/vendor/pear-pear.php.net/XML_Util:/var/www/vendor/phing/phing/classes:.:/usr/local/lib/php:/var/www/test/../vendor/phing/phing/classes:/var/www/test/fixtures/bookstore/build/classes') in /var/www/runtime/lib/Propel.php on line 302

Fatal error: Uncaught PropelException: Unable to open configuration file: '/var/www/test/tools/helpers/bookstore/../../../fixtures/bookstore/build/conf/bookstore-conf.php' in /var/www/runtime/lib/Propel.php:304
Stack trace:
#0 /var/www/runtime/lib/Propel.php(338): Propel::configure('/var/www/test/t...')
#1 /var/www/test/tools/helpers/bookstore/BookstoreTestBase.php(13): Propel::init('/var/www/test/t...')
#2 /var/www/test/testsuite/generator/behavior/AddClassBehaviorTest.php(11): require_once('/var/www/test/t...')
#3 /var/www/vendor/phpunit/phpunit/src/Util/Fileloader.php(56): include_once('/var/www/test/t...')
#4 /var/www/vendor/phpunit/phpunit/src/Util/Fileloader.php(38): PHPUnit_Util_Fileloader::load('/var/www/test/t...')
#5 /var/www/vendor/phpunit/phpunit/src/Framework/TestSuite.php(310): PHPUnit_Util_Fileloader::checkAndLoad('/var/www/test/t...')
#6 /var/www/vendor/phpunit/phpunit/src/Framework/TestSuite.php(384): PHPUnit_Framework_TestSuite->addTestFile('/var/www/test/t...')
#7 /var/www/vendor/phpunit/phpunit/src/Uti in /var/www/runtime/lib/Propel.php on line 304
```